WooCommerce/Classes/Tools/Notices/Notice.swift

```diff
@@ -11,6 +11,10 @@ struct Notice {
     ///
     let title: String
 
+    /// An optional subtitle for the notice
+    ///
+    let subtitle: String?
+
     /// An optional subtitle for the notice
     ///
     let message: String?
```

Comment:
ℹ️ We might want to consider clarifying the difference between `subtitle` and `message`. Both have the same descriptions. Maybe a description indicating how they are displayed.

Reply:
Great suggestion! Updated in https://github.com/woocommerce/woocommerce-ios/pull/5299/commits/eea115f296e1ab8e5f8203d12e2c479518542469 based on the [Apple documentation](https://developer.apple.com/documentation/usernotifications/unmutablenotificationcontent/1649873-subtitle) on these fields